src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Text.cs

```diff
@@ -150,14 +150,9 @@ internal static void ValidateFormatArgs(string name, CheckTypesContext checkType
                 errors.EnsureError(DocumentErrorSeverity.Moderate, args[1], TexlStrings.ErrNotSupportedFormat_Func, name);
                 isValid = false;
             }
-            else
+            else if (!BuiltInEnums.DateTimeFormatEnum.FormulaType._type.Accepts(argTypes[1], exact: true, useLegacyDateTimeAccepts: false, usePowerFxV1CompatibilityRules: checkTypesContext.Features.PowerFxV1CompatibilityRules))
             {
-                if (BuiltInEnums.DateTimeFormatEnum.FormulaType._type.Accepts(argTypes[1], exact: true, useLegacyDateTimeAccepts: false, usePowerFxV1CompatibilityRules: checkTypesContext.Features.PowerFxV1CompatibilityRules))
-                {
-                    // Coerce enum values to string
-                    CollectionUtils.Add(ref nodeToCoercedTypeMap, args[1], DType.String);
```

Comment:
Is this coercion not needed? Double check running this against canvas, if possible.

Reply:
I add flag StronglyTypedBuiltinEnums check and put back code so canvas can go with old flow.